Task 6/script.js

```diff
@@ -19,3 +19,26 @@ const users = [
   { id: '8', name: 'Simon Peterson', age: 30 },
   { id: '9', name: 'Daniel Cane', age: 51 },
 ];
+
+let sumAge = 0;
+
+const getUserAverageAge = (array) => {
+
+  array.forEach(user => {
+    sumAge += user.age;
+  });
+
+  countAverage = sumAge / users.length
```

Comment:
Labai bloga praktika kurti kintamuosius be let/const raktažodžio. Tokiu būdu jūs sukūrėte naują property globaliame objekte, konkrečiau - window objekte. Pabandykite pažiūrėti konsolėje kaip atrodo jūsų window objektas ir suraskite ten countAverage. Taip nedarykite, raktažodžiai tuo ir naudingi, kad kintamieji scope'inami objektuose pagal jų poziciją, taip išvengsite globalaus scope šiukšlinimo.

Pastaba dėl kintamojo pavadinimo: kintamieji su pradžia count/get/create/get/calc imponuoja liepiamąją nuosaką, kas būdinga funkcijoms, kurios turi kažką padaryti. Šiuo atveju tai tiesiog kintamasis, saugantis skaičių. Tai pačiu, geriau būtų perkelti visą Math objekto logiką čia pat, nei laikyti console.log() parametre.